m5_forecasting/data_processor.py

```
@@ -0,0 +1,156 @@
+        desired_column_order = ['unique_id', 'ds', 'y', 'sell_price', 'num_events', 'cat_id', 'dept_id', 'state_id', 'store_id',
+                                'day_of_week', 'is_weekend', 'day_of_month', 'week_of_month', 'month', 'week_num_year', 'year']
+        self.sales_data = self.sales_data[desired_column_order]
+
+    def split_data(self):
+        """Splits the processed data into train and test sets based on the horizon from the config."""
+        if self.processor is None:
+            raise ValueError("Data has not been processed. Please call 'prcoess_data' before 'split_data")
+        horizon = self.horizon
+        train_list = []
+        test_list = []
+        
```

Ruff

Blank line contains whitespace

Remove whitespace from blank line

(W293)